Foundation/CGFloat.swift

```diff
@@ -279,33 +292,28 @@ extension CGFloat : BinaryFloatingPoint {
     }
 
     @_transparent
-    public var magnitude: CGFloat {
-        return CGFloat(Swift.abs(native))
-    }
-
-    @_transparent
     public mutating func negate() {
         native.negate()
     }
 
     @_transparent
-    public mutating func add(_ other: CGFloat) {
-        native.add(other.native)
+    public static func +=(_ lhs: inout CGFloat, _ rhs: CGFloat) {
```

Comment:
Added the missing methods here:  https://github.com/apple/swift/pull/3796/commits/192c11217fe0c1a1821004b34434ffe523875707